protocol/requests/create_account.go

```diff
@@ -12,27 +12,43 @@ var ErrCreateAccountInvalidBackupDisabledDataDir = errors.New("create-account: i
 var ErrCreateAccountInvalidLogFilePath = errors.New("create-account: invalid log file path")
 
 type CreateAccount struct {
-	DisplayName        string `json:"displayName"`
-	Password           string `json:"password"`
-	ImagePath          string `json:"imagePath"`
-	CustomizationColor string `json:"customizationColor"`
 	// RootKeystoreDir is the directory where keys are stored
 	RootKeystoreDir string `json:"rootKeystoreDir"`
```

Comment:
not relate to your PR, can we remove this field now? because we calculate it in the backend.

Reply:
is that already merged or it's in a different pr?

Reply:
> then rootKeystoreDir is ignored on status-go side and hardcoded "keystore" is used https://github.com/status-im/status-go/blob/9151aa7f042ca0fe3bf9f0c3201a122ee452926d/api/geth_backend.go#L738 

referenced from discord, i thought you saw the message 🙂

Reply:
I'd do it in a different PR, this one is already quite a few changes :)